Mathlib/Analysis/SpecialFunctions/ImproperIntegrals.lean

```diff
@@ -35,6 +35,13 @@ theorem integrableOn_exp_Iic (c : ℝ) : IntegrableOn exp (Iic c) := by
   simp_rw [norm_of_nonneg (exp_pos _).le, integral_exp, sub_le_self_iff]
   exact (exp_pos _).le
 
+theorem integrableOn_exp_neg_Ioi (c : ℝ) : IntegrableOn (fun (x : ℝ) => exp (-x)) (Ioi c) := by
```

Comment:
I think you should prove an analogue of [MeasureTheory.Integrable.comp_neg](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Group/Integral.html#MeasureTheory.Integrable.comp_neg) for `IntegrableOn` and general sets, and likewise for everything else in that file. Then just use it here.

And why not `Ici c`?

Reply:
Yeah I'll try extracting a `IntegrableOn.comp_neg`-like theorem. As for why not `Ici`, it seems that most theorems are only stated for `Ioi` and `Iic`, like existing `integral_exp...` in this file. But if we think it is better to have all 4 versions, I certainly can add them

Reply:
Sorry, I meant to take out that "Why not ...?" but forgot. Please note my other comment about adding an explanation for why we don't include the others.